src/component/ManageMeetings/ManageMeetings.js

```diff
@@ -1,14 +1,69 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { Button, Container, Row, Col } from 'react-bootstrap';
+import './ManageMeetings.css';
 
 const ManageMeetings = () => {
     
-
-  
-    return <div>
         <h1>ManageMeetings</h1>
+
+    return <div className="container">
+
+    <Container>
+
+        <Row>
+        <h1><b>Manage Meetings</b></h1>
+        </Row>
+        
+<br></br>
+
+        <Row>
+            <Col>
+                <ul class="nav nav-tabs">
+                <li class="nav-item">
+                <a class="nav-link active" href="#">Student</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Tutor</a>
+                </li>
+                </ul>
+            </Col>
+            <Col></Col>
+            <Col></Col>
```

Comment:
what do these empty <Col>s do?

Reply:
They're for spacing so it divides the grid further, to make those Student/Tutor tabs further from the Search +Schedule Meeting button on the same row. I'm sure there's a better way of doing this, but this seems to work.